source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+
+      .. code-block:: console
+
+        C:\> CertUtil -hashfile ./clear-<version>-installer.iso.xz | findstr -v file
+
+      Compare the output with the original checksum to make sure they match.
+
+Uncompress the image
+====================
+
+* For Linux distros (an installer ISO image is used as an example):
+
+  .. code-block:: console
+
+    $ unxz clear-<version>-installer.iso.xz
+
+* For Windows:
+
+  Use `7zip`_ to uncompress it.
+
+Method 1: Fresh installation of |CL| into a new VM 
+==================================================
+
+The general process for performing a fresh installation of |CL| into a new VM 
+is as follows (with expanded details below):
+
+* Create a new VM and configure its settings
+* Attach the installer ISO to it
+* Install |CL|
+* Detach the installer ISO
+* Power off the VM
+* Enable EFI boot support
+* Power on the VM
+
+Create and configure a new VM 
+*****************************
+
+#.  Start the `VMware Workstation Player` app.
+#.  On the home screen:
+
+    * Click `Create a New Virtual Machine`.
+
+      |vmware-player-01|
+
+      Figure 1: VMware Workstation 14 Player - Create a new virtual machine
+   
+#.  On the `Welcome to the New Virtual Machine Wizard` step:
+
+    * Select `Installer disc image file (iso)`.
+    * Click `Browse` and select the uncompressed |CL| installer ISO. 
+    * Click `Next`.
+
+      |vmware-player-02|
+
+      Figure 2: VMware Workstation 14 Player - Select |CL| installer ISO   
+
+#.  On the `Select a Guest Operating System` step:
+
+    * Set `Guest operating system` to `Linux`.
+    * Set `Version` to `Other Linux 3.x or later kernel 64-bit`.
+    * Click `Next`.
+
+      |vmware-player-03|
+
+      Figure 3: VMware Workstation 14 Player - Select guest operating system type
+
+#.  On the `Name the Virtual Machine` step:
+
+    * Give it a name.
+    * Click `Next`.
+
+      |vmware-player-04|
+
+      Figure 4: VMware Workstation 14 Player - Name virtual machine
+
+#.  On the `Specify Disk Capacity` step:
+
+    * Set the desired disk size.
+    * Click `Next`.
+
+      |vmware-player-05|
+
+      Figure 5: VMware Workstation 14 Player - Set disk capacity
+
+#.  On the `Ready to Create Virtual Machine` step:
+
+    * Click `Customize Hardware...`
+    
+      |vmware-player-06|
+
+      Figure 6: VMware Workstation 14 Player - Customize hardware
+
+    * Select `Processors` > `Virtualization engine` and 
+      check `Virtualize Intel VT-x/EPT or AMD-V/RVI`.
+
+      |vmware-player-07|
+
+      Figure 7: VMware Workstation 14 Player - Set virtualization engine option
+
+    * Click `Close`.
+
+    * Click `Finish`.
+
+Install |CL| into the new VM
+****************************
+
+#.  Select the newly created VM and click `Play virtual machine`.  
+
+    |vmware-player-08|
+
+    Figure 8: VMware Workstation 14 Player - Power on virtual machine
+
+#.  Follow the :ref:`bare-metal-install` (starting at the `Install Clear Linux 
```

Comment:
You can add a tag in bare-metal-install.rst just prior to the section you want readers to jump to. Reference that tag in this doc and readers that click will have the tag at the top of their page.